aimmo-game/simulation/avatar/avatar_manager.py

```diff
@@ -23,8 +23,39 @@ def remove_avatar(self, user_id):
 
     @property
     def avatars(self):
-        return self.avatars_by_id.viewvalues()
+        return list(self.avatars_by_id.viewvalues())
 
     @property
     def active_avatars(self):
         return [player for player in self.avatars]
+
+    def players_update(self):
+        """
+        To be called on a required update of players in the front-end of the game. This is a wrapper for player_list.
+        :return: A dictionary that holds the list generated by player_list.
+        """
+
+        def _player_list():
```

Comment:
the word list here and below is misleading. Here it's a method that lists one player's data.

Reply:
It returns a list of all players data, all appended, not just one. Then it's wrapped in a dictionary by `players_update`

Reply:
I'll change it to `_generate_player_list()` instead, like in the other class perhaps?